php-packages/testing/src/integration/TestCase.php

```diff
@@ -46,6 +46,8 @@ protected function tearDown(): void
     protected function app()
     {
         if (is_null($this->app)) {
+            $this->config('env', 'testing');
```

Comment:
Why was this added / why wasn't it needed before?

Reply:
since we now implement the laravel `ApplicationContract` which has the  `environement` method that can be called internally in illuminate components, we might as well also make sure the value is correct in testing. (that's the gist of the changes in this PR)